app/domain/errors.rb

```diff
@@ -292,16 +292,21 @@ module AuthnAzure
           code: "CONJ00050E"
         )
 
-        ClaimInInvalidFormat = ::Util::TrackableErrorClass.new(
-          msg:  "xms_mirid claim has been received in an invalid format. Reason: {0}",
+        TokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+          msg:  "Field '{0-field-name}' not found or empty in token",
           code: "CONJ00051E"
         )
 
-        TokenFieldNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
-          msg:  "Field '{0-field-name}' not found or empty in token",
+        XmsMiridParseError = ::Util::TrackableErrorClass.new(
```

Comment:
`XmsMiridParseError ` please add this to the log issue for TWs
cyberark/conjur-docs#742
Create a new section "Added logs" to the end so Shuli knows which new ones to review

Reply:
I added these 2 as a PR comment